ga4gh/datamodel/references.py

```diff
@@ -5,3 +5,56 @@
 from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
+
+import os
+import glob
+
+import pysam
+
+import ga4gh.protocol as protocol
+
+
+class ReferenceSet(object):
+    """
+    TODO document.
+    """
+    def __init__(self, id_, dataDir):
+        self._id = id_
```

Comment:
Definitely agree, we will certainly want a common superclass here. I'm not sure quite what it'll look like yet though, so I thought it best to just get something simple that works first.